src/Symfony/Component/Notifier/Notification/Notification.php

```diff
@@ -36,21 +36,19 @@ class Notification
     public const IMPORTANCE_MEDIUM = 'medium';
     public const IMPORTANCE_LOW = 'low';
 
-    private array $channels = [];
-    private string $subject = '';
     private string $content = '';
     private string $emoji = '';
     private ?FlattenException $exception = null;
     private string $exceptionAsString = '';
     private string $importance = self::IMPORTANCE_HIGH;
 
-    /**
-     * @param list<string> $channels
-     */
-    public function __construct(string $subject = '', array $channels = [])
-    {
-        $this->subject = $subject;
-        $this->channels = $channels;
+    public function __construct(
+        private string $subject = '',
+        /**
+         * @var list<string>
+         */
+        private array $channels = [],
+    ) {
```

Comment:
```suggestion
    /**
     * @param list<string> $channels
     */
    public function __construct(
        private string $subject = '',
        private array $channels = [],
    ) {
```

Reply:
Same discussion in https://github.com/symfony/symfony/pull/53241#discussion_r1437880895